.circleci/config.yml

```diff
@@ -1,28 +1,47 @@
 version: 2
 
 defaults: &defaults
-  working_directory: ~/project/yarn
   docker:
-    - image: yarnpkg/dev:latest
+    - image: node:8
+  working_directory: ~/project/yarn
 
-default_filters: &default_filters
-  tags:
-    only: /^v[0-9]+\.[0-9]+\.[0-9]+$/
+attach_workspace: &attach_workspace
+  attach_workspace:
+      at: ~/project
 
-restore_node_modules: &restore_node_modules
+restore_cache: &restore_cache
   restore_cache:
     name: Restore node_modules cache
     keys:
       - v1-node-{{ .Branch }}-{{ checksum "yarn.lock" }}
       - v1-node-{{ .Branch }}-
       - v1-node-
 
+test_steps: &test_steps
+  steps:
+    - *attach_workspace
+    - *restore_cache
+    - run:
+        name: Tests
+        command: |
+          if [ "$CIRCLE_BRANCH" == 'master' ]; then
+            ./scripts/set-dev-version.js
+          fi;
+          yarn build
```

Comment:
We can possibly move this step into `install` and save a few more seconds per test run.